src/Bridge/Faker/Provider/StoryblokProvider.php

```diff
@@ -46,6 +46,7 @@ public function storiesResponse(array $overrides = []): array
             'stories' => [],
             'cv' => $this->generator->randomNumber(),
             'rels' => [],
+            'rel_uuids' => [],
```

Comment:
this property is not always returned by the API and is added only when the limit of 50 resolved relations is crossed